bcmserver.c

```diff
@@ -179,9 +180,11 @@ int main(void)
 	saddr.sin_addr.s_addr = htonl(INADDR_ANY);
 	saddr.sin_port = htons(PORT);
 
+	struct timespec r;
 	while(bind(sl,(struct sockaddr*)&saddr, sizeof(saddr)) < 0) {
 		printf(".");fflush(NULL);
-		usleep(100000);
+		struct timespec f = {0, 100 * 1000 * 1000};
+		nanosleep(&f, &r);
```

Comment:
You're not using `&r`, so please use `NULL` intead.

Reply:
I added r as this is in a loop.

Reply:
But `r` is not used, why do you need it?

Reply:
Fixed.